src/main/java/ch/njol/skript/classes/PatternedParser.java

```diff
@@ -0,0 +1,23 @@
+package ch.njol.skript.classes;
+
+import ch.njol.util.StringUtils;
+
+/**
+ * Subclass of {@link Parser} for getting literal patterns.
```

Comment:
i would appreciate more explanation on what the class actually is. What's a literal pattern? Something more like "a parser that is based off of a set list of valid patterns to match" would be better. Also `subclass of x` is already described by `extends x` so no need to say the same thing twice.